include/openthread/border_routing.h

```diff
@@ -118,6 +118,21 @@ typedef struct otBorderRoutingPrefixTableEntry
     uint32_t                   mPreferredLifetime;   ///< Preferred lifetime of the on-link prefix when `mIsOnLink`.
 } otBorderRoutingPrefixTableEntry;
 
+/**
+ * Represents a group of info when platform generated RA message processed.
+ *
+ * The entries in the discovered table track the Prefix/Route Info Options in the received Router Advertisement messages
+ * from other routers on the infrastructure link.
+ *
+ */
+typedef struct otPdProcessedRaInfo
+{
+    bool    mRequirePD;               ///< Indicates whether the platform require PD
+    int64_t mNumPlatformRaReceived;   ///< The number of platform generated RA handled by ApplyPlatfromGeneratedRa
+    int64_t mNumPlatformPIOProcessed; ///< The number of PIO processed for adding OMR prefixes
+    int64_t mLastPlatformRaTimestamp; ///< The timestamp of last processed RA message
```

Comment:
How is the timestamp represented? Why is it `int64`?

Reply:
Seems like the code is using the `TimerMilli::GetNow()` for this to keep track of last processed RA. Note that this is available within OT core only (and platform). User of OT APIs don't know the notion of `GetNow()` (and don't need to).

General model use in other OT APIs is to represent the timestampls as "milliseconds (or seconds) since last processed RA" (which is then calculated when the API is called to get the info).  As example, you can check mMsecSinceLastUpdate` in `otBorderRoutingPrefixTableEntry`. Suggest using the same approach here.

Reply:
Thanks, updated